docs/concepts/dev-setup.md

```diff
@@ -72,16 +72,7 @@ Installation:
 
 This is our help tool that will ease the contribution process. It will help you to generate a [Pack](../packs/packs-format), maintain your files, and validate them before committing to the branch.
 
-To install **demisto-sdk** using `pipx`
-```bash
-pipx install demisto-sdk --force
-```
-
-To check the you have the latest version of the SDK, run:
-
-```bash
-demisto-sdk --version
-```
```

Comment:
I think you can leave these instructions here they are not really an SDK documentation

Reply:
I'm not sure the pipx is still relevant at all - will consult with @samuelFain as well

Reply:
While pipx uses pip under the hood to install Python packages (meaning it can install demisto-sdk), I'm not familiar with any current documentation or use-case related to demisto-sdk that recommends using `pipx` over `pip`.
@dorschw WDYT?

Reply:
pip is fine